File: csw-logging/csw-logging-client/src/main/java/csw/logging/client/javadsl/JLogAppenderBuilders.java
package csw.logging.client.javadsl;

import csw.logging.client.appenders.FileAppender$;
import csw.logging.client.appenders.LogAppenderBuilder;
import csw.logging.client.appenders.StdOutAppender$;

/**
 * Helper class for Java to get the handle of csw provided appenders
 */
public class JLogAppenderBuilders {

    /**
     * Represents StdOut as appender for logs
     */
    public static final LogAppenderBuilder StdOutAppender = StdOutAppender$.MODULE$;

    /**
     * Represents file as appender for logs
     */
    public static final LogAppenderBuilder FileAppender = FileAppender$.MODULE$;
}

File: csw-logging/csw-logging-client/src/main/scala/csw/logging/client/javadsl/JLogAppenderBuilders.scala
package csw.logging.client.javadsl

import csw.logging.client.appenders
import csw.logging.client.appenders.LogAppenderBuilder

/**
 * Helper class for Java to get the handle of csw provided appenders
 */
object JLogAppenderBuilders {

  /**
   * Represents StdOut as appender for logs
   */
  val StdOutAppender: LogAppenderBuilder = appenders.StdOutAppender

  /**
   * Represents file as appender for logs
   */
  val FileAppender: LogAppenderBuilder = appenders.FileAppender
}
